docs/migrationV4.md

```diff
@@ -0,0 +1,67 @@
+---
+id: migrationv4
```

Comment:
I checked and this does not seem to be published to the docs site somehow? Perhaps it is just a timing thing or part of the release process delayed, but I thought I would mention it

Reply:
Oops! I was looking for the wrong thing. It's published, I see it. Sorry.